tools/generators/lib/PBXProj/src/Identifiers.swift

```diff
@@ -6,45 +6,74 @@ import CryptoKit
 /// Since the various `PBXProj` partial generators need to work independently
 /// from each other, they need help to be able to generate unique identifiers.
 /// `Identifiers` is used to generate these identifiers.
+///
+/// The identifiers aren't all hashes. Grouping information is also encoded into
+/// them. The first two characters indicate that it's a global element (`FF`), a
+/// file or group element (`FE`), or a generator shard (`00`-`FD`).
+///
+/// - Global elements (which start with `FF`) are then classified as belonging
+///   to the project as a whole (`00`) or the `BazelDependencies` target
+///   (`01`). For both of them, their associated elements (e.g. configuration
```

Comment:
I don't immediately understand this portion.

Is it `FF...01` & `FF...00`, `FF01...` & `FF00...`, or something else?

Reply:
`FF00...` and `FF01...`.

I'm 100% open to rewording. This applies to all of your comments.